policy-descriptor/policy-schema.yml

```diff
@@ -0,0 +1,113 @@
+---
+$schema: "http://json-schema.org/draft-04/schema#"
+title: "Policy Descriptor Schema"
+version: 0.1
+description: "The core schema for SONATA policy descriptors."
+
+##
+## Some definitions used later on.
+##
+definitions:
+  time_units:
+    enum:
+      - "s"   # seconds
+      - "m"   # minutes
+      - "h"   # hours
+      - "d"   # days
+  aggregatin_function:
+    enum:
+      - "avg"   # average
+      - "min"   # min
+      - "max"   # max
+  monitoring:
+    type: "object"
+    properties:
+      name:
+       description: "The name of the parameter to monitor. The name has to be supported by the service platform or the FSM."
+       type: "string"
+       pattern: "^[A-Za-z-_]+$"
+      unit:
+       description: "The unit used to monitor (or represent) the parameter."
+       oneOf:
+         - $ref: "#/definitions/general_units"
+         - $ref: "#/definitions/memory_units"
```

Comment:
I guess, you need to include these definitions at the top too in order to reference them